.changeset/cool-waves-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@google-labs/breadboard": patch
```

Comment:
should be minor, since removing a file is breaking

Reply:
```suggestion
"@google-labs/breadboard": minor
```